apps/sensenet/src/components/SearchButton.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from 'react'
+import { useRepository } from '@sensenet/hooks-react'
```

Comment:
Please follow the naming convention for files. We use kebab-case for new files now.